server/server.go

```diff
@@ -1502,6 +1502,7 @@ func (s *Server) v1DataPost(w http.ResponseWriter, r *http.Request) {
 			result.Explanation, err = types.NewTraceV1(*buf, pretty)
 			if err != nil {
 				writer.ErrorAuto(w, err)
+				return
```

Comment:
This is unrelated to the change, something I've stumbled upon. I don't believe there should be any `writer.ErrorAuto` calls that aren't followed by a `return`.